openapi-generator/src/test/kotlin/org/jellyfin/openapi/builder/api/ApiNameBuilderTests.kt

```diff
@@ -0,0 +1,37 @@
+package org.jellyfin.openapi.builder.api
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class ApiNameBuilderTests {
+	@Test
+	fun `Names end with Api`() {
+		val instance = ApiNameBuilder()
+
+		assertTrue { instance.build("test").endsWith("Api") }
+		assertTrue { instance.build("api").endsWith("Api") }
+		assertTrue { instance.build("testApi").endsWith("Api") }
+		assertTrue { instance.build("fourSeemsLikeEnoughAsserts").endsWith("Api") }
```

Comment:
:eyes:

Reply:
This is one of the parts I wrote not long before opening the PR when reviewing everything myself in a few hours. :)